challenges/KICKin-Rollin-Challenge.tex

```diff
@@ -0,0 +1,46 @@
+% !TeX root = ../SPL-Challenges.tex
+% !TeX spellcheck = en_US
+\section{KICKin\textquotesingle \& Rollin\textquotesingle Challenge}
+
+The objective of this challenge is for a robot to successfully kick a rolling ball from a ramp into a designated goal area. 
+Each team participating will record three attempts spread accross three days of the competition.
+An attempt consist of three kick attempts. 
+
+\subsection{Field Setup}
+
+The challenge will take place on a standard SPL field with its official marking. 
+The ramp will be positioned perpendicularly adjacent to the halfway line, aligned with a virtual line extending from the extremity of the center circle on the opponent's side.
+The ramp can be positioned at varying distances from the robots to modify the difficulty.
+The participating robot can be positioned at the team’s discretion along the virtual line between the center circle and the penalty mark.
+The robot must have one foot on each side of this virtual line.
+
+\subsubsection{Ramp}
+
+The ramp design will be available on the SPL website.
+The ramp may have an adjustable angle to modify the difficulty.
+
+
+\subsection{Challenge Procedure}
+
+At the scheduled time, every participating must regroup on the designated field. 
+The ramp will be set up by the technical committee and will remain the same for all attempts executed this day.
+Each team will perform their three kicks successively.
+Teams may bring as many robots as needed, but they will not be allowed to deploy code between attempts.
+
+\subsection{Scoring}
+
+Each kick will receive a score based on the location of the ball.
+These points are non-cumulative: 
+
+\begin{itemize}
+	\item 0 point for a missed kick or deflected ball.
```

Comment:
A deflected ball needs more precise definition to differentiate it from a kicked ball. Does deflected mean that the robot foot did not move? If the foot was moving (e.g. for a kick) and connected with the ball but didn't connect with the front of the foot (hence deflected), surely the team should still get some points.